src/cli/commands/init.ts

```diff
@@ -0,0 +1,31 @@
+import { prompt } from "inquirer";
+import { Configuration, ProjectType, Runner } from "../../engine";
+
+export const initCommand = () =>
+  prompt([
+    {
+      name: "project",
+      message: "What do you want to do?",
+      type: "list",
+      choices: [
+        { name: "Create an express project", value: ProjectType.Express },
+        {
+          name: "Create a serverless project",
+          value: ProjectType.Serverless,
+        },
+      ],
+    },
+    {
+      name: "name",
+      message: "What is your project name?",
+      type: "input",
+    },
+    {
+      name: "modules",
+      message: "Which modules would you like to launch?",
+      type: "checkbox",
+      choices: ["redis"],
```

Comment:
I guess we should use enums here for fields like `modules`, `project`, `redis`.

Reply:
Yes, you are right